.gdn/.gdntsa

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Replace references of this file with ".config/tsaoptions.json". Then delete this file. You may do it in another PR.